shenyu-admin/src/main/java/org/apache/shenyu/admin/model/query/FieldQuery.java

```diff
@@ -0,0 +1,150 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shenyu.admin.model.query;
+
+import org.apache.shenyu.admin.model.page.PageParameter;
+
+import java.util.Objects;
+
+public class FieldQuery {
+
+    /**
+     * the fieldName.
+     */
+    private String name;
+
+    /**
+     * the field desc.
+     */
+    private String fieldDesc;
+
+
+    /**
+     * the ext.
+     */
+    private String ext;
```

Comment:
please remove `ext` condition, it is appropriate to use ES, but ES is too complex, so this field is abandoned directly.

Reply:
done